client/src/game/ui/drawUI.ts

```diff
@@ -1395,15 +1399,13 @@ const drawTimers = () => {
     listening: true,
   });
   globals.layers.timer.add(globals.elements.timer1 as any);
-  globals.elements.timer1.on('click', (event) => {
+  function timerClick() {
```

Comment:
always prefer `const timerClick = () => {` to `function timerClick() {`  UNLESS you need `this`, and it doesn't look like you are using `this` here